phoenicis-javafx/src/main/java/org/phoenicis/javafx/components/application/control/ApplicationView.java

```diff
@@ -0,0 +1,174 @@
+package org.phoenicis.javafx.components.application.control;
+
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
+import org.phoenicis.javafx.components.application.skin.ApplicationViewSkin;
+import org.phoenicis.javafx.components.common.control.PhoenicisView;
+import org.phoenicis.javafx.components.common.widgets.utils.ListWidgetType;
+import org.phoenicis.javafx.settings.JavaFxSettingsManager;
+import org.phoenicis.javafx.themes.ThemeManager;
+import org.phoenicis.javafx.views.mainwindow.apps.ApplicationFilter;
+import org.phoenicis.repository.dto.ApplicationDTO;
+import org.phoenicis.repository.dto.CategoryDTO;
+import org.phoenicis.scripts.interpreter.ScriptInterpreter;
+
+/**
+ * The component shown inside the Phoenicis "Applications" tab
+ */
+public class ApplicationView extends PhoenicisView<ApplicationView, ApplicationViewSkin> {
```

Comment:
Why did you rename `ApplicationsView` to `ApplicationView`? It shows multiple applications.

Reply:
Done